lockup/src/internal.rs

```diff
@@ -97,4 +97,12 @@ impl LockupContract {
             env::panic(b"No NEAR Foundation account is specified in the contract");
         }
     }
+
+    pub fn assert_owner(&self) {
```

Comment:
Maybe it makes sense to have `assert_foundation`, by analogy.

Reply:
There `assert_called_by_foundation` above.